Documentation/ceph-csi-drivers.md

```diff
@@ -19,6 +19,8 @@ For documentation on consuming the storage:
 * RBD: See the [Block Storage](ceph-block.md) topic
 * CephFS: See the [Shared Filesystem](ceph-filesystem.md) topic
 
+Both drivers also support the creation of static PV and static PVC from existing RBD image/CephFS volume. Refer to [static PVC](https://github.com/ceph/ceph-csi/blob/devel/docs/static-pvc.md) for more information.
```

Comment:
How about creating a header for this section? This will make it easier to link when people have questions. Thanks!
```
## Static Provisioning
```